lib/rubygems/command.rb

```diff
@@ -351,7 +352,7 @@ def when_invoked(&block)
   # options in output.  See `gem help list` for an example.
 
   def add_option(*opts, &handler) # :yields: value, options
-    group_name = Symbol === opts.first ? opts.shift : :options
+    group_name = Symbol == opts.first.class ? opts.shift : :options
```

Comment:
This is only a stylistic change, right? I tried enabling rubocop's [`Style/CaseEquality`](https://rubocop.readthedocs.io/en/latest/cops_style/#stylecaseequality) cop, and we currently have 109 offenses in rubygems code base, so maybe we can remove the change from this PR and discuss enabling that cop separately.

Reply:
Sounds good 👍